substrate-query-framework/index-builder/src/HydraServer.ts

```diff
@@ -0,0 +1,35 @@
+import * as bodyParser from 'body-parser';
+import { Server } from '@overnightjs/core';
+import { Logger } from '@overnightjs/logger';
+
+export class HydraServer extends Server {
```

Comment:
Should we rename it to HydraLogServer? and I didn't understand why you are introducing this? is it good to have running another process with the indexer?

Reply:
I have deleted this file, it was committed there by mistake. Indeed all the server stuff will be located at the top-level package index-server